src/Faker/Provider/Color.php

```diff
@@ -91,6 +91,14 @@ public static function rgbCssColor()
     }
 
     /**
+     * @example 'rgba(0,255,122,0.8)'
+     */
+    public static function rgbaCssColor()
```

Comment:
I tried to respect the naming convention of the function "rgbCssColor". That's why I think this is the most appropriate. Don't you think so ?